lib/litani.py

```diff
@@ -58,6 +58,22 @@ def is_expired(self):
 
 
 
+class ReportRenderingCompletion:
+    """This class is to mark the completion of the report rendering process"""
+
+    def __init__(self, path: pathlib.Path):
```

Comment:
I don't think this class should take a `path` argument. The argument makes it easier to write unit tests because you can then pass a mock object, but this class is so simple that it doesn't need tests. At the same time, the only `path` that ever gets passed is the `cache_dir`, so I suggest that you call `get_cache_dir` from `__init__`. That will simplify both call sites.